_topics/en/redundant-overpayments.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Redundant overpayments
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - Stuckless payments
+  - Boomerang payments
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightning Network
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: Stuckless payments
+      link: https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-June/002029.html
+
+    - title: Boomerang
+      link: https://arxiv.org/pdf/1910.01834.pdf
+
+    # Spear (not available publicly AFAICT): https://dl.acm.org/doi/10.1145/3479722.3480997
+
+## Optional.  Each entry requires "title" and "url".  May also use "feature:
+## true" to bold entry and "date"
+optech_mentions:
+  - title: Stuckless payments proposed
+    url: /en/newsletters/2019/07/03/#stuckless-payments
+
+  - title: "LN summit 2021: stuckless payments"
+    url: /en/newsletters/2021/11/10/#ln-summit-2021-notes
+
+  - title: "Speeding up payment delivery with algorithms and stuckless payments"
+    url: /en/newsletters/2022/03/23/#payment-delivery-algorithm-update
+
+  - title: "Presentation summary: Boomerang"
+    url: /en/newsletters/2020/02/26/#boomerang-redundancy-improves-latency-and-throughput-in-payment-channel-networks
+
+  - title: "Using redundant overpayments instead of a quality-of-service flag"
+    url: /en/newsletters/2023/02/22/#ln-quality-of-service-flag
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: Simplified multipath payments
+    link: topic multipath payments
+
+  - title: Atomic multipath payments
+    link: topic amp
+
+## Optional.  Force the display (true) or non-display (false) of stub
+## topic notice.  Default is to display if the page.content is below a
+## threshold word count
+#stub: false
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Redundant overpayments** are LN payments split into parts where the
+  spender sends a greater amount and more parts than necessary to pay
+  the receiver's invoice.
+
+---
+Even if some of the parts fail to arrive at the receiver's node on the
+first try due to forwarding failures, enough of the other parts may
+arrive to allow the receiver to claim their invoiced amount.  Protocol
+features prevent the receiver from claiming more than the invoiced
+amount.  Compared to sending an exact amount, over paying initially can
```

Comment:
```suggestion
amount.  Compared to sending an exact amount, overpaying initially can
```

Reply:
@LarryRuane   I'm curious why this instance of single-word "overpaying" is suggested for separation into two words when the rest of the instances in the file are not mentioned?